virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc

```diff
@@ -8,10 +8,6 @@ toc::[]
 You can configure a Single Root I/O Virtualization (SR-IOV) device for virtual machines in your cluster.
 This process is similar but not identical to configuring an SR-IOV device for {product-title}.
 
-[NOTE]
-====
-Live migration is supported for virtual machines that are attached to an SR-IOV network interface only if the `sriovLiveMigration` feature gate is enabled in the HyperConverged Cluster custom resource (CR). When the `spec.featureGates.sriovLiveMigration` field is set to `true`, the `virt-launcher` pod runs with the `SYS_RESOURCE` capability. This might degrade its security.
-====
 
 == Prerequisites
```

Comment:
While you're in this file, could you please add an anchor ID for each `==` level heading?